db/models.py

```diff
@@ -1,6 +1,11 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
 
+from settings import AUTH_USER_MODEL
```

Comment:
The import statement `from settings import AUTH_USER_MODEL` is incorrect. It should be `from django.conf import settings` and then use `settings.AUTH_USER_MODEL` in the ForeignKey definition.

Reply:
from django.contrib.auth import get_user_model

User = get_user_model()

https://stackoverflow.com/questions/54710952/django-importing-auth-user-model